packages/compiler/src/core/formatter-fs.ts

```diff
@@ -3,35 +3,52 @@ import { globby } from "globby";
 import { resolveConfig } from "prettier";
 import { PrettierParserError } from "../formatter/parser.js";
 import { checkFormatTypeSpec, formatTypeSpec } from "./formatter.js";
+import { Diagnostic, NoTarget } from "./index.js";
+import { createDiagnostic } from "./messages.js";
 import { normalizePath } from "./path-utils.js";
 
 export interface TypeSpecFormatOptions {
   exclude?: string[];
   debug?: boolean;
 }
 
+export interface TypeSpecFormatResult {
+  formattedFiles: string[];
```

Comment:
can you just add a doc commnent saying this is the path to the formatted files and if it is absolute or relative to the pattern asked. Feel like its something always missing on docs and you have to run the funciton once to know.

Reply:
Will do